core/src/modules/broadcastHelper.ts

```diff
@@ -11,5 +11,7 @@ export async function broadcastModel<T>(
       model: await instance.apiData(),
       verb,
     });
-  } catch {}
+  } catch {
+    // Ignore error
+  }
```

Comment:
At the basic, we get the [no-empty](https://eslint.org/docs/rules/no-empty) rule that discourages empty blocks. The easiest way to fix it is to put a comment.